src/user/user.controller.ts

```diff
@@ -27,4 +28,23 @@ export class UserController {
       await this.userService.testSubscription()
     ).data;
   }
+
+  @Patch('make-admin/:userId')
+  @ApiOperation({ summary: 'Convert An Existing User To An Admin' })
+  async makeAdmin(
+    @Param('userId') id: string,
+  ): Promise<Ok<User>> {
+    const admin =  await this.userService.makeUserAnAdmin(id)
+    return ZaLaResponse.Ok(admin, 'Admin Created', 201)
+  }
+
+  @Get('all-registered-users')
+  @ApiOperation({ summary: 'Get All Registered Users' })
+  async getAllUsers(
+    @Query('date-from') dateFrom: string
+  ): Promise<Ok<User[]>> {
+    const allUsers =  await this.userService.getallUsers(dateFrom)

```

Comment:
Is the function returning all registered users between a date range or only unverified users between the specified date range? 
Because the endpoint description is saying something different

Reply:
yea i've changed it.. forgot to change it to "isEmailVerified: true" after running tests.. but i've just corrected it